examples/passport-login/src/controllers/facebook.controller.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-passport-login
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  get,
+  RestBindings,
+  Response,
+  RequestWithSession,
+} from '@loopback/rest';
+import {inject, intercept} from '@loopback/core';
+import {UserProfile} from '@loopback/security';
+
+/**
+ * Login controller for facebook
+ */
+export class FacebookController {
+  constructor(
+  ) {}
+
+  @intercept('passport-init-mw', 'passport-session-mw', 'passport-facebook')
```

Comment:
It is possible to define a custom decorator such as:

```
export function facebookLogin() {
  return intercept('passport-init-mw', 'passport-session-mw', 'passport-facebook');
}
```

Then we can use `@facebookLogin()`.